src/main/java/ua.horobets/hw17/Main.java

```diff
@@ -0,0 +1,146 @@
+package ua.horobets.hw17;
```

Comment:
```
Configuration configuration = new Configuration();
configuration.configure();
SessionFactory sessionFactory = configuration.buildSessionFactory();
```
This should be done only once. Move it before invoking your task in main method.
SessionFactory object is one for all application life-cycle.

Very good solution also would be:
In some utility class (SessionUtil) you need to initialize your factory and has some method getSession() which returns EntityManager/Session for working with database